### PR TITLE
Fix social media links #2560

### PR DESCRIPTION
**Pull request by khalilbenihoud**
Following @IAmMorrow work on the social media links feature, I fixed some bugs:

- Updated the Ledger Academy icon 
- The "Help" icon is now between the settings and the discreet mode
- Added the close icon/action to the modal
- Wording changes

fix-header

fix-modal

### CODE REVIEW

**Comment by khalilbenihoud** [comment]
Added wording changes following @dasilvarosa  suggestion!

**Comment by valpinkman** [comment]
🔥

File: src/renderer/components/TopBar/index.js [hunk 1]
```diff
@@ -114,9 +114,14 @@ const TopBar = () => {
             <Box justifyContent="center">
               <Bar />
             </Box>
-            <Tooltip content={t("settings.title")} placement="bottom">
-              <ItemContainer id="topbar-settings-button" isInteractive onClick={navigateToSettings}>
-                <IconSettings size={16} />
+            {/*  */}
```
Comment (valpinkman) [inline src/renderer/components/TopBar/index.js:117]: remove empty comment

**Comment by christhep** [comment]
Crypto & Security 101 does not fit in the window. Let's change to Learn crypto

**Comment by khalilbenihoud** [comment]
> Learn crypto

Done!

**Pull request merged**